{"name": "test_get_title", "status": "broken", "statusDetails": {"message": "AttributeError: 'TestLogin' object has no attribute 'driver'", "trace": "self = <login_test.TestLogin object at 0x000001FAE0576100>\n\n    @pytest.fixture(scope='function',autouse=True)\n    def setup_login(self):\n>       self.login  =Login_Page(self.driver)\nE       AttributeError: 'TestLogin' object has no attribute 'driver'\n\ntests\\login_test.py:16: AttributeError"}, "description": "Check the Login page", "start": 1730126547732, "stop": 1730126547732, "uuid": "53d85a50-16c2-4161-9e7d-ce98319870ce", "historyId": "87bbdde710f2a5ac5e4f116ce1b79cb6", "testCaseId": "87bbdde710f2a5ac5e4f116ce1b79cb6", "fullName": "tests.login_test.TestLogin#test_get_title", "labels": [{"name": "severity", "value": "blocker"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "login_test"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "MSI"}, {"name": "thread", "value": "29152-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.login_test"}]}